CMakeLists.txt

```diff
@@ -15,9 +15,19 @@
 cmake_minimum_required(VERSION 3.25)
 cmake_policy(SET CMP0079 NEW) # Allow target_link_libraries() in subdirs
 
+include(src/cmake/feature-flags.cmake)
 include(src/cmake/environment-utils.cmake)
 is_running_in_ci(IS_RUNNING_IN_CI)
 
+# A feature flag with a dependency.
+feature_flag(APPLEVZ_ENABLED "AppleVZ backend" APPLE)
```

Comment:
I'm not 100% sure, but the impression was that all the feature flags (an possibly feature flag specific cmake changes) would live in a separate file/module.

@xmkg would probably be able to provide a more knowledgeable opinion than I could.

Reply:
Yeah, our spec says we'll put these in `feature_flags.cmake` at the root of the tree. I just wanted to throw together a quick example here that I'll remove before merging this (though I suppose I could have done that in the proper file...)